script.js

```diff
@@ -0,0 +1,229 @@
+
+const TEXTAREA = document.createElement('textarea');
```

Comment:
I'll suggest to create elements in separate function

Reply:
fix in commit 963f612